GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs

```diff
@@ -388,12 +387,11 @@ private void InitializeComponent()
             this.MostRecentLB.Location = new System.Drawing.Point(0, 19);
             this.MostRecentLB.MultiSelect = false;
             this.MostRecentLB.Name = "MostRecentLB";
-            this.MostRecentLB.OwnerDraw = true;
+            this.MostRecentLB.Enabled = false;
```

Comment:
why have `LessRecentLB` and `MostRecentLB` become disabled?

Reply:
These ListViews previously had owner-draw code to remove the appearance of items being selected. I think just making the controls disabled does just as good a job - what do you think?

*Old:*
![2018-02-20 gitextensions old](https://user-images.githubusercontent.com/2142963/36421716-bdce545a-168d-11e8-85cf-3d55810ffea4.png)

*New:*
![2018-02-20 gitextensions new](https://user-images.githubusercontent.com/2142963/36421715-bd704d10-168d-11e8-99c6-54681f4a3b8d.png)

Reply:
not quite, now it is impossible to anchor or remote items

![image](https://user-images.githubusercontent.com/4403806/36422733-7be51ef8-1691-11e8-87a4-95f280d8575d.png)

Reply:
Oops, somehow missed that functionality entirely. I've reverted the code for these two ListView controls.

Reply:
Yes, the UI is not particularly obvious....